README.md

```diff
@@ -5,21 +5,19 @@ Rust-Lightning
 [![Documentation](https://img.shields.io/static/v1?logo=read-the-docs&label=docs.rs&message=lightning&color=informational)](https://docs.rs/lightning/)
 [![Safety Dance](https://img.shields.io/badge/unsafe-forbidden-success.svg)](https://github.com/rust-secure-code/safety-dance/)
 
-`rust-lightning` is a Bitcoin Lightning library written in Rust. The main crate,
-`lightning`, does not handle networking, persistence, or any other I/O. Thus,
-it is runtime-agnostic, but users must implement basic networking logic, chain
-interactions, and disk storage. More information is available in the `About`
-section.
+[LDK](https://lightningdevkit.org)/`rust-lightning` is a highly performant and flexible 
+implementation of the Lightning Network protocol.
+
+The main crate, `lightning` is runtime-agnostic. Data persistence, chain interactions, and
+networking can be provided by LDK's [sample modules](#crates), or custom implementations
+may be provided by the user.
```

Comment:
Since we are using "you" and "your", say "you" of "the user" and use the active voice:

"or you may provide your own custom implementations"